src/test/java/org/apache/commons/lang3/ObjectUtilsTest.java

```diff
@@ -586,6 +586,23 @@ public void testIsEmpty() {
         assertFalse(ObjectUtils.isEmpty(NON_EMPTY_MAP));
     }
 
+    /**
+     * Test for {@link ObjectUtils#isArray(Object)}.
+     */
+    @Test
+    public void testArray() {
+        assertFalse(ObjectUtils.isArray(null));
+        assertFalse(ObjectUtils.isArray(""));
+        assertFalse(ObjectUtils.isArray("abg"));
+        assertFalse(ObjectUtils.isArray(123));
+
+        assertTrue(ObjectUtils.isArray(NON_EMPTY_ARRAY));
+        assertTrue(ObjectUtils.isArray(new int[] {}));
```

Comment:
Might as well test with all of our own arrays like the `EMPTY_*` arrays.

Reply:
Done